modules/database/src/adapters/DatabaseAdapter.ts

```diff
@@ -8,6 +8,16 @@ export abstract class DatabaseAdapter<T extends SchemaAdapter<any>> {
   registeredSchemas: Map<string, ConduitSchema>;
   models?: { [name: string]: T };
 
+  /**
+   * Checks if the database has already been connected with Conduit
+   */
+  abstract isConduitDB(): Promise<boolean>;
```

Comment:
Convert to proper camelCase.

Reply:
Check whether any of `Core`'s schemas exist instead.
Introspection could fail half-way, whereas `Core` only gets a chance to register any of its schemas after the initial introspection is finalized.